en/faq/appengine-deployment.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Google App Engine Deployment
+description: How to deploy Nuxt.js on Google App Engine?
+---
+
+# How to deploy on Google App Engine?
+
+Deploying to [Google App Engine](https://cloud.google.com/appengine/) is a fast and easy solution for hosting your universal Nuxt application on Google's Cloud Services.
+
+In this guide, we build the application locally and then simply upload the entire project folder to Google App Engine. After the upload, Google App Engine will automatically start the `start` script in our package.json and your app will be available immediately.
+
+## Getting Started
+
+Make sure you have a Google Cloud Account, a project and an empty Google App Engine app set up on https://cloud.google.com/appengine/. Furthermore, make sure to download and install the Cloud SDK (CLI) from Google as explained [here](https://cloud.google.com/sdk//) and log into your Google Cloud Account.
```

Comment:
Double `/` on the "here" link

Reply:
Also, making a link out of the bare appengine reference would be appreciated :relaxed:

Reply:
Totally agree with that, thx for noticing! :)

Reply:
Fixed.